index.d.ts

```diff
@@ -219,7 +221,7 @@ declare module 'ical-generator' {
       type(): attendeeType;
       type(type: attendeeType): ICalAttendee;
       delegatedTo(): string | ICalAttendee;
-      delegatedTo(attendee: string | ICalAttendee);
+      delegatedTo(attendee: string | ICalAttendee): ICalAttendee;
```

Comment:
@Wicusiu added a return type, and I aligned it to match the other overloads. See: https://github.com/sebbo2002/ical-generator/pull/128#pullrequestreview-182538188